packages/perseus/src/widgets/iframe/iframe.tsx

```diff
@@ -90,8 +97,7 @@ class Iframe extends React.Component<Props> implements Widget {
         });
     };
 
-    change: (...args: ReadonlyArray<unknown>) => any = (...args) => {
-        // @ts-expect-error - TS2345 - Argument of type 'readonly unknown[]' is not assignable to parameter of type 'any[]'.
+    change: (...args: ChangeParams) => any = (...args) => {
```

Comment:
It's also possible to give the method an overall type (ChangeFn) like several other widgets, but that introduces `null | undefined` as potential return types, which I think we are steering away from. This defines the parameter types to allow us to remove the `ts-expect-error`.